lib/api/CAnomalyJob.cc

```diff
@@ -358,6 +359,17 @@ bool CAnomalyJob::handleControlMessage(const std::string& controlMessage) {
         break;
     case 'w':
         this->processPersistControlMessage(controlMessage.substr(1));
+        break;
+    case 'z':
+        LOG_INFO(<< "Received control message '" << controlMessage << "'");
```

Comment:
I should have noticed this before, but this is going to be unacceptably spammy in production. If you want to leave it as INFO for a day or two to observe the effect that’s OK. But if you do that we need another PR before feature freeze that sets it to TRACE. Or set it to TRACE in this PR if you prefer.